src/components/DownloadFile.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+/**
+ * Download file link.
+ * 
+ * @param props {object}
+ * @param props.children {string|null} - Optional link text
+ * @param props.data {object} - Data to be saved
+ * @param props.filename {string}
+ */
+class DownloadFile extends React.Component {
```

Comment:
My bad - I intended to have the disclaimer as it was later on, where it contained a trigger to the feedback form. I must have been pretty tired last night. I'll find the correct commit and link it - it just uses the same stuff that the other 'Feedback' button uses.